### PR TITLE
feat(TreeView): removed selectable modifier for penta

### LINKED ISSUE #10010: Consume core updates: Tree view  ( Follow up to issue)

### PR DESCRIPTION
**Pull request by wise-king-sullyman**
**What**: Closes #10010 

**Additional issues**:

### CODE REVIEW

**Review by andrew-ronaldson — APPROVED** [review]
Looks good! Thanks

**Review by kmcfaul — APPROVED** [review]

File: packages/react-core/src/components/TreeView/TreeViewListItem.tsx [hunk 1]
```diff
@@ -208,7 +208,6 @@ const TreeViewListItemBase: React.FunctionComponent<TreeViewListItemProps> = ({
             <Component
               className={css(
                 styles.treeViewNode,
-                children && (isSelectable || hasCheckbox) && 'pf-m-selectable',
```
Comment (thatblindgeye) [inline packages/react-core/src/components/TreeView/TreeViewListItem.tsx:211]: Might be worth keeping this in as a string, similar to what was done to [Calendar month in the core bump PR](https://github.com/patternfly/patternfly-react/pull/10081/files#diff-d158e2d7f9e30b2ef3b1527ae50fae669b00830d5d5cbec6c1a74fd1e0721b86R262). Worth getting feedback from core on this, though.

Comment (wise-king-sullyman) [reply packages/react-core/src/components/TreeView/TreeViewListItem.tsx:211]: @mcoker said to rip it out

**Review by thatblindgeye — APPROVED** [review]

**Review by mcoker — APPROVED** [review]
🚀

**Review by tlabaj — APPROVED** [review]

**Pull request merged**